tests/e2e/specs/endorse-owner.spec.js

```diff
@@ -0,0 +1,39 @@
+describe(
+  "Endorse Owner",
+  {
+    retries: {
+      runMode: 5,
+      openMode: 0,
+    },
+  },
+  () => {
+    it("should endorse owner successfully", () => {
+      cy.visit("/verify");
+      cy.get("input[type=file]").attachFile("ebl-endorse-owner.json");
+      cy.get("[data-testid='asset-title-owner']").should("be.visible");
+      cy.get("[data-testid='asset-title-holder']").should("be.visible");
+
+      // START - approve application once after connect to wallet, subsequent tests no longer need
```

Comment:
does this still hold true for cypress? so endorse owner spec is the first test it will run and how is the order determined?

Reply:
so previously transfer-holder is the first test, but in synpress the order of the tests is by alphabetical order, so this as of now this is the 1st test it will run so we will need this approve application after connect to wallet here. 
additionally, in future if there is another file that should run before this file, we will need to update the test cases accordingly.